src/api/db/migrate/20161122121211_move_comments_data.rb

```diff
@@ -3,11 +3,14 @@ def change
     ActiveRecord::Base.transaction do
       Comment.inheritance_column = nil
       Comment.find_each(batch_size: 5000) do |comment|
+        next unless comment
```

Comment:
because we had nil objects as comment.
Better continue if there is nothing to convert ... or shall we clean up here actually?